csbuild/tools/cpp_compilers/gcc_cpp_compiler.py

```diff
@@ -121,7 +121,7 @@ def _getDebugArgs(self):
 	def _getOptimizationArgs(self):
 		arg = {
 			OptimizationLevel.Size: "s",
-			OptimizationLevel.Speed: "fast",
+			OptimizationLevel.Speed: "2",
```

Comment:
Not part of this PR but I feel like these level names are confusing. We should revisit this at some point, because I have a hard time working out in my head whether "Speed" or "Max" is supposed to be faster.

Reply:
Agreed, I was never a huge fan of it either, but it's just been like that for lack of a better option. It's roughly close to what compilers traditionally offer in terms of optimization options, but it's not universally applicable for every compiler. As to not knowing which one to choose as the faster option, that's not solely our problem. Some compilers are known for producing faster code when optimizing for executable size. Not really a moral to that story, just a fun little quirk.